views/providers.py

```diff
@@ -200,8 +200,7 @@ def delete_hawkular_provider(self):
 
         # Verify if the provider is deleted from the provider list.
 
-        assert ui_utils(self.web_session).waitForElementOnPage(By.XPATH, "//a[@title='{}']".format(
-            self.web_session.HAWKULAR_PROVIDER_NAME), 120, exist=False)
+
```

Comment:
I do not see why providers.py has this new assert - should not have any changes in providers.py. Please remove this file from the PR/Branch.